articles/ros_node_idl.md

```diff
@@ -113,36 +113,26 @@ Another is the fact that, as soon as the node is running, RCL itself (or another
 How do upstream packages specify their interface requirements?
 Through a high-level description of all the actions, parameters, services and topics, provided or required, by each node within the package.
 
-The package interface is defined in a separate [XML][xml_wiki] file and exported from the `package.xml` using [REP 149's export mechanism][rep149_export], thereby avoiding pollution of the package manifest.
+The package interface is defined in a separate [XML][xml_wiki] file, thereby avoiding pollution of the package manifest.
+The XML file is exported from the package's `CMakeLists.txt` file, or from the `setup.py` file for a Python package.
+The export is supported by a `CMake` macro, `nodl_export_interface_description_file`, which installs the file to the current package's share folder as well as installs the required `ament` index resources needed for the interface description to be discovered later.
```

Comment:
I think we've shortened that now, haven't we?

Reply:
Slightly - `nodl_export_node_description_file `